code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -319,6 +319,22 @@
 		return
 	if(M.has_bane(BANE_SALT))
 		M.mind.disrupt_spells(-200)
+	if(method=INGEST && is_species(M, /datum/species/squid))
```

Comment:
`=` sets a var equal to something, you want `==` which checks equality.

Reply:
Fuck me. Beginner (pro gamer) move